app-office/texmacs/texmacs-1.99.12.recipe

```diff
@@ -0,0 +1,102 @@
+SUMMARY="A WYSIWIG editor for beautiful structured technical documents"
+DESCRIPTION="TeXmacs provides a unified and user friendly framework for \
+editing structured documents with different types of content: text, \
+mathematics, graphics, interactive content, slides, etc.
+Its rendering engine uses high-quality typesetting algorithms so as to \
+produce professionally looking documents, which can either be printed out \
+or presented from a laptop."
+HOMEPAGE="http://www.texmacs.org/"
+COPYRIGHT="Free Software Foundation"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/mgubi/texmacs/archive/texmacs-$portVersion-haiku-qt5.tar.gz"
+CHECKSUM_SHA256="607fa4db4676b78780f7af086feec2e2332dc18a371d79fa50d640de6db10b43"
+SOURCE_DIR="texmacs-texmacs-$portVersion-haiku-qt5/src"
+PATCHES="texmacs-$portVersion.patchset"
+ADDITIONAL_FILES="texmacs.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	texmacs$secondaryArchSuffix = $portVersion
+	app:TeXmacs
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libQt5Concurrent$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Svg$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libguile$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libgmp$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libltdl$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libiconv$secondaryArchSuffix
+	devel:libQt5Concurrent$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Svg$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	devel:libguile$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libgmp$secondaryArchSuffix
+	devel:libltdl$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:automake
+	cmd:find
+	cmd:gcc$secondaryArchSuffix
+	cmd:gettext$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:sed
+	cmd:guile == 1.8.8
+	"
+
+BUILD()
+{
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	mkdir $appsDir
+	mv $binDir/fig2ps $libExecDir/TeXmacs/bin/fig2ps
+	mv $libExecDir/TeXmacs/bin/texmacs.bin $appsDir/TeXmacs
+	rm -rf $binDir
+	rm -rf $libExecDir/TeXmacs/lib
```

Comment:
For some reason this deletes the whole $libExecDir, so maybe we can just leave it out (checked with the content in HaikuDepot)?
```diff
- 	rm -rf $libExecDir/TeXmacs/lib
```
PS, you can check a rebuild with "hp texmacs -F" (this skips the build and jumps to "INSTALL")

Reply:
It works fine on my machine, lib/TeXmacs/bin is in the package the recipe produces. I do not see any problem.

Reply:
I checked with package list

Reply:
I've also checked in HaikuDepot and the package seems ok (lib/TeXmacs/bin and its content are present)

Reply:
For some reason this is failing on gcc2h then? 
https://ibb.co/Tw1jF9c

Reply:
There hasn't been any change to the patchset aside from the recipe yes?

Reply:
No change. There are no local changes in my git repository.

Reply:
what is the value of $libExecDir in your machine? I do not anyway understand why lib/TeXmacs/bin disappears.

Reply:
I think it's due to "rm -rf $binDir", reverted that to what you initially had "rm $binDir/texmacs" and that seems to work, but leaves an empty $binDir behind

Reply:
What about `rm -rf $binDir/texmacs` and `rmdir $binDir$ ? I've pushed a change right now.